effective-image-guide/readme.md

```diff
@@ -0,0 +1,36 @@
+>**The goal of this guide:** to explore what makes an image effective in relation to an article.
+
+# Sciencing the $@%* out of images
+
+When used correctly, effective visuals can communicate a message very powerfully. Let's look at some of the science behind visual content and what makes an image effective.
+
+## First things first: Why visual content is critical
+
+There are vast amounts of telling stats about how big visual content has become and how it’s changing the way we all approach marketing. Check out the [research by Buffer's Kevan Lee](https://blog.bufferapp.com/a-complete-guide-to-creating-awesome-visual-content), or [Design & Dev's Tom More](http://thenextweb.com/dd/2014/05/21/importance-visual-content-deliver-effectively/). [Search Engine Watch's Tereza Litsa](https://searchenginewatch.com/2016/04/21/understanding-our-love-of-visual-content/) also has a lot to say about visual content. Much of the following information is also gleaned from her research. 
+
+## What makes an effective, relevant image?
+
+There are a number of interesting scientific and psychological explanations for why certain images resonate with people and others don't:
+
+* **Appeal**
+
+Appeal refers to natural attraction towards visual content and how it manages to attract even those with short attention spans. We might not be able to explain why we love an image because [our conscious thought hasn’t yet caught up with our subconscious](https://en.wikipedia.org/wiki/Empirical_relationship). Deeply emotional reactions are some of the strongest connections we can make to visual content. They occur from the part of the brain that is also related to our [survival instincts](http://adventure.howstuffworks.com/survival/wilderness/wired-for-survival1.htm), which means that the reaction may be more direct and intense.
+
+Visual content should indicate the emotion for which it aims. This could be accomplished abstractly with color, metaphors, or irony, for example. Or it could be achieved concretely with associations which elicit the "right" feeling (safety, food, positivity, love...).
```

Comment:
I want to indicate here that "right" is highly specific to the context, since there is no such thing as a "right feeling". Found this at http://www.grammarbook.com/punctuation/quotes.asp:

Rule 8a. Quotation marks are often used with technical terms, terms used in an unusual way, or other expressions that vary from standard usage.

Examples:
It's an oil-extraction method known as "fracking."
He did some "experimenting" in his college days.
I had a visit from my "friend" the tax man.

Rule 8b. Never use single quotation marks in sentences like the previous three.

Incorrect: I had a visit from my 'friend' the tax man.

The single quotation marks in the above sentence are intended to send a message to the reader that friend is being used in a special way: in this case, sarcastically. Avoid this invalid usage. Single quotation marks are valid only within a quotation, as per Rule 7, above.